src/common-principles.md

```diff
@@ -653,6 +653,23 @@ for more information.
 
 ## The Inheritance Principle
 
+In some circumstances, there can be multiple data files for which
+all or a subset of the relevant metadata is precisely equivalent.
+Where this occurs,
+it may be preferable to define those metadata *only once*,
+and be placed on the filesystem in such a way that those files
+are deemed to be *applicable* to each relevant data file individually,
+but *not* be erroneously associated with other data files
+to which the metadata contained within are not applicable.
+The Inheritance Principle defines a systematized set of rules
+to determine which metadata files to associate with which data files.
+Further, due to the prospect of an individual data file having
```

Comment:
I'm not a native speaker, but I would prefer a shorter sentence:

_Further, because multiple metadata files may apply to an individual data, the Principle defines the *order of precedence* of such metadata files contents._

Reply:
Adopted in 8154a6b648878634b6581e25bdd98336aca70c61.